app/Console/Commands/UpdateEmailCohortGat7004.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\EmailTemplate;
+use Illuminate\Console\Command;
+
+class UpdateEmailCohortGat7004 extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:update-email-cohort-gat7004';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Cohort Discovery update email will.expire template to remove renewal button';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
```

Comment:
please update `EmailTemplateSeeder`

Reply:
Ok - we're keeping the seeder up to date with the changes via commands? Happy to.

Reply:
command is for deployment and seeder in this moment is for tests